homeassistant/components/nextcloud/manifest.json

```diff
@@ -5,5 +5,5 @@
   "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/nextcloud",
   "iot_class": "cloud_polling",
-  "requirements": ["nextcloudmonitor==1.4.0"]
+  "requirements": ["nextcloudmonitor==1.5.0"]
```

Comment:
please use the [`script/gen_requirements_all.py`](https://developers.home-assistant.io/docs/development_testing#adding-new-dependencies-to-test-environment) when bumping dependencies or even better use our [pre-commit](https://developers.home-assistant.io/docs/development_testing) hook or our [VS Code devcontainer](https://developers.home-assistant.io/docs/development_environment#developing-with-visual-studio-code--devcontainer)

Reply:
you still need to run at least the `script/gen_requirements_all.py` script to update all relevant requirement files (_just bump the version in the `manifest.json` than run this script_).
Meanwhile please also update the docs [nextcloud.markdown](https://github.com/home-assistant/home-assistant.io/blob/current/source/_integrations/nextcloud.markdown) and add the `update` platform to the `ha_platforms` and `Update` to h`a_category` ... than link the docs PR here on the initial post under "Link to documentation pull request:"

Reply:
Bumping dependencies should be done in a separate PR anyways.

Reply:
i think in this case it is ok, because it is needed for the new feature